packages/generators/init-generator.ts

```diff
@@ -59,6 +59,7 @@ export default class InitGenerator extends Generator {
 			config: {
 				configName: this.isProd ? "prod" : "config",
 				topScope: [],
+				// TODO migrate tslint
```

Comment:
This is now done, so you can rebase from master. I think that comment has been deleted there.

Reply:
Yeah, I left that comment as a reminder for myself :) I'm waiting to complete my work on the generator, just to avoid rebasing multiple times